src/NetscriptEvaluator.ts

```diff
@@ -4,20 +4,21 @@ import { ScriptDeath } from "./Netscript/ScriptDeath";
 import { WorkerScript } from "./Netscript/WorkerScript";
 import { NetscriptContext } from "./Netscript/APIWrapper";
 
-export function netscriptDelay(time: number, workerScript: WorkerScript): Promise<void> {
+export function netscriptDelay(fnName: string, time: number, workerScript: WorkerScript): Promise<void> {
```

Comment:
if your going to modify netscriptDelay to require the function name you should match the existing functions that do so and put a wrapped version in the context helpers to pass in the function name automatically

Reply:
Standardized in #3967 which includes all these changes. Closing this PR since everything should still go in on the other one.